src/templates/TemplateRetriever.ts

```diff
@@ -92,10 +92,10 @@ export abstract class TemplateRetriever {
 
     protected getBackupVersion(runtime: ProjectRuntime): string {
```

Comment:
This isn't needed anymore, right?

Reply:
Yeah it still is. This is the version of the backup templates shipped with the extension.